Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -38,23 +38,54 @@
             "name": "PostDeploymentAction",
             "displayName": "Post Deployment Action",
             "isExpanded": false,
-            "visibleRule": "WebAppKind != \"\""
+            "visibleRule": "ConnectionType = AzureRM || WebAppKind != \"\""
```

Comment:
Should it be &&

Reply:
And I think we should remove all WebAppKind != "". @vincentdass to confirm

Reply:
done